saleor/product/models.py

```diff
@@ -52,6 +52,10 @@ def get_absolute_url(self):
             'product:category',
             kwargs={'slug': self.slug, 'category_id': self.id})
 
+    @property
+    def is_published(self):
```

Comment:
I think this is unnecessary, we could just use a `getattr` with a fallback to `True` if it's not present.

Reply:
Sorry, I didn't understand Your proposal. My idea was to have `is_published` property in `Page`, `Category` and `Collection` to have possibility to check published status of `MenuItem` as `linked_object.is_published`

Reply:
Instead of using `linked_object.is_published` you can use `getattr(linked_object, 'is_published', True)`

Which will return True if `is_published` is not present on the model.

Reply:
Got it, thanks!